### DISCUSSION TITLE
求助：怎么单独启动server服务

### DISCUSSION THREAD

**Discussion by Metasymbols** [AlistGo/alist#5152] [Q&A]
不需要前端web，单独提供api

**Comment by anwen-anyi** [comment]
直接启动后端不行吗？
```go
go run main.go
```

**Reply by hywan9** [reply]
go run main.go是这样的
<img width="1471" alt="image" src="https://github.com/alist-org/alist/assets/78184805/6fbecb69-d390-4717-a602-75524c401a7b">
go run main.go server 没有前端又没办法启动

**Reply by anwen-anyi** [reply]
> go run main.go是这样的
> 
> <img width="1471" alt="image" src="https://github.com/alist-org/alist/assets/78184805/6fbecb69-d390-4717-a602-75524c401a7b">
> 
> go run main.go server 没有前端又没办法启动

你得把编译好的前端dist文件移动到指定文件

**Comment by hywan9** [comment]
我是想问能不能前后端都单独运行: (

安稳 ***@***.***>于2023年10月31日 周二12:24写道：
